hbase-it/src/test/java/org/apache/hadoop/hbase/ipc/IntegrationTestRpcClient.java

```diff
@@ -66,7 +67,8 @@ public IntegrationTestRpcClient() {
     conf = HBaseConfiguration.create();
   }
 
-  protected AbstractRpcClient<?> createRpcClient(Configuration conf, boolean isSyncClient) {
+  protected AbstractRpcClient<?> createRpcClient(Configuration conf, boolean isSyncClient)
```

Comment:
Do we still need to throw SSLContextException here?

Reply:
This one too.